### DISCUSSION TITLE
Support for Ungoogled Chromium / Firefox instead of Chrome

### DISCUSSION THREAD

**Discussion by JIGASHI** [FlareSolverr/FlareSolverr#735] [Q&A]
Is there any way to use portable install of ungoogled chromium or Firefox instead of chrome, if so, how?

**Comment by ngosang** [comment]
Sorry but it will cause issues.
If you don't want to install Chome, you can use the Docker images or the pre-compiled packages.
https://github.com/FlareSolverr/FlareSolverr/releases/tag/v3.1.1
